.gitignore

```diff
@@ -5,6 +5,7 @@
 
 .idea/
 .gradle/
+.kotlin/
```

Comment:
occasionally .kotlin files are left hanging around following a test run this prevents them being committed